subiquitycore/utils.py

```diff
@@ -176,3 +176,10 @@ def disable_subiquity():
                  "snap.subiquity.subiquity-service.service",
                  "serial-subiquity@*.service"])
     return
+
+
+def matching_dicts(items, **kw):
```

Comment:
suggestions:
* `**criteria` instead of `**kw`
* `items: Sequence[Dict[Any, Any]]` so it's more obvious for the caller what to pass

Reply:
Done!

Reply:
Done!